yarn.lock

```diff
@@ -135,7 +133,7 @@ assert-plus@^1.0.0:
   version "1.0.0"
   resolved "https://registry.yarnpkg.com/assert-plus/-/assert-plus-1.0.0.tgz#f12e0f3c5d77b0b1cdd9146942e4e96c1e4dd525"
 
-async@1.x, async@^1.4.0, async@^1.4.2:
+async@^1.4.0, async@^1.4.2, async@1.x:
```

Comment:
Don’t update `yarn.lock` if your fix doesn’t need it. “One PR — one issue” is good idea :D.